playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -100,8 +100,8 @@
   hosts: oo_masters_to_config:oo_nodes_to_config
   vars:
     target_version: "{{ '1.2' if deployment_type == 'origin' else '3.1.1.900' }}"
-    openshift_docker_hosted_registry_insecure: True
-    openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
+    docker_hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(True) }}"
```

Comment:
This wasn't working as is? I'd expect these to get passed down to openshift_docker as `openshift_docker_hosted_registry_insecure` and `openshift_docker_hosted_registry_network` where they're actually used. I think just switching from hardcoded to the defaulting is enough.

Need opinion of someone more versed in ansible than myself though. @detiber / @abutcher ?